### ISSUE TITLE
Address bar does not accept a manually entered URI

### ISSUE THREAD

**Issue by sporcina** [brave/browser-laptop#4623]
**Did you search for similar issues before submitting this one?**
Yes

**Describe the issue you encountered:**
I am unable to manually type in a URI through the address bar (e.g. "www.google.ca", "http://www.google.com", or "google").  As soon as I hit 'enter' the address defaults back to "https://www.brave.com/".

**Expected behavior:**
Should present the URI in the address bar and navigate to that site.

- Platform (Win7, 8, 10? macOS? Linux distro?):
Windows 10 Pro
Version 1607
OS Build 14393.222

- Brave Version:
0.12.4

- Steps to reproduce:
    1. Launch Brave via desktop shortcut
    2. Type in "http://www.google.ca" in the address bar
    3. Hit 'enter' key

- Screenshot if needed:
None required.  It looks just like it does when you start up Brave.

- Any related issues:
No

**Comment by Sh1d0w** [comment]
Can you see if this is not duplicate of an issue I have reported recently https://github.com/brave/browser-laptop/issues/4219 ?

**Comment by sporcina** [comment]
No it is not.  #4219 is about entering a url, but navigating to a different one.  Mine is that no manually typed url is accepted;  the address bar goes blank.  I recently had to wipe my machine and clean install Windows.  After installing Brave again the address bar works correctly now.  I'm wondering if a older version of Brave introduced the problem that was never resolved in newer patches.  I do know that when I uninstalled Brave and reinstalled it, the problem persisted.
